dask-gateway-server/requirements.txt

```diff
@@ -0,0 +1,9 @@
+# These are dependencies for installing the dask-gateway-server package.
+#
+# NOTE: changes to the dependencies here must also be reflected in
+#       ../dev-environment.yaml
+#
+aiohttp
```

Comment:
Version bounds?

Reply:
The versions pinning seen here are unchanged from being extracted from setup.py, do you suggest we put bounds like <4 here or similar?

I've understood it as a bad practice to constrain the versions of a python package without knowing ahead of time its a problem and are used to just declaring upper bounds when its know to cause issues that are to be fixed but aren't yet fixed.

Reply:
I imagine there are versions that just don't work (any more); but sometimes it's useful to guard against unknown future breakages. But since this was copied, perhaps this isn't the time. Also it's hard to do thorough research on compatible versions.

Reply:
Ah, yes lower bounds would be good to maintain. I think it's a very relevant improvement to aim for. I opened #539 about this!